analytics/src/test/java/com/segment/analytics/AnalyticsTest.java

```diff
@@ -796,19 +796,6 @@ protected boolean matchesSafely(TrackPayload payload) {
                         payload.properties().getInt("build", -1) == 100;
                   }
                 }));
-    verify(integration)
```

Comment:
why was this removed?

Reply:
Both were removed as `ApplicationOpened` is no longer called during `trackApplicationLifecycleEventsInstalled`. It is now called in `onActivityResumed` and depending if it's a cold start or not, the correct Properties are passed.

Reply:
Got it - shouldn't this test be moved somewhere else then?

Reply:
I see it is moved, never mind!